cognite_toolkit/_cdf_tk/commands/_base.py

```diff
@@ -56,10 +57,10 @@ def run(self, execute: Callable[..., Any], *args: Any, **kwargs: Any) -> Any:
             self._track_command("success")
         return result
 
-    def warn(self, warning: ToolkitWarning) -> None:
+    def warn(self, warning: ToolkitWarning, include_timestamp: bool = False, console: Console | None = None) -> None:
```

Comment:
When using the status from rich, we need to specify the console to get warnings to print out appropriately.